CMSIS/NN/Include/arm_nnfunctions.h

```diff
@@ -975,7 +975,8 @@ int32_t arm_depthwise_conv_s8_opt_get_buffer_size(const uint16_t input_ch,
    * @param[in]       output_activation_min        for clamping
    * @param[in]       output_activation_max        for clamping
    * @param[in]       vec_buffer                   pointer to buffer space used for optimization and is necessary
```

Comment:
How big should vec_buffer be?

Reply:
@gmiodice It is mentioned couple of lines below as 'col_dim * sizeof(q15_t) bytes'